modules/dhcp_isc/dhcpd_conf_parser.rb

```diff
@@ -0,0 +1,7 @@
+require 'rsec'
+
+module ::Proxy::DHCP::ISC
+  class DhcpdConfParser < Parslet::Parser
+
```

Comment:
Nit pick, I'd perhaps do `class DhcpdConfParser < Parslet::Parser; end` but not sure if Rubocop will like it.